frontend/appflowy_tauri/src/appflowy_app/components/layout/NavigationPanel/PageItem.hooks.ts

```diff
@@ -17,12 +25,23 @@ export const usePageEvents = (page: IPage) => {
     closePopup();
   };
 
-  const changePageTitle = (newTitle: string) => {
+  const changePageTitle = async (newTitle: string) => {
+    await FolderEventUpdateView(
```

Comment:
Please use `ViewBackendService` and move the logic to the renamePage handler.

Reply:
I've already updated this file. You should "select changes from all commits"